onnxruntime/core/providers/coreml/builders/impl/activation_op_builder.cc

```diff
@@ -95,6 +95,19 @@ Status ActivationOpBuilder::AddToModelBuilderImpl(ModelBuilder& model_builder,
   } else if (op_type == "PRelu") {
     auto* prelu = layer->mutable_activation()->mutable_prelu();
     ORT_RETURN_IF_ERROR(AddPReluWeight(model_builder, node, logger, *prelu));
+  } else if (op_type == "LeakyRelu") {
+    const auto& attributes = node.GetAttributes();
```

Comment:
```suggestion
    NodeAttrHelper helper(node);
    const auto alpha = helper.Get("alpha", 0.01f);
```
using 0.01 as the default value for alpha - LeakyRelu